contrib/docker/docker-compose.yml

```diff
@@ -52,6 +52,7 @@ services:
       # If using production, comment development volume below
       #SUPERSET_ENV: production
       SUPERSET_ENV: development
+    user: root:root
```

Comment:
Since the contrib/docker folder already has a .env file in it, it would likely be best if the user was an environmental variable declared in that file rather than hard-coded in the docker-compose file, IMHO.  

For example, in the .env file add two lines:
```
UID=1000
GID=1000
```
...and then in the docker-compose file use indirect references:

```yaml
    user: ${UID}:${GID}
```

Reply:
Well, I think it's because docker runs on root. I don't use root user neither, but docker daemon runs with root and bound files need root user to access them.